### PR TITLE
Fix settings.lua to use new additions / changes to DarkRP base settings.lua

### PR DESCRIPTION
**Pull request by liondadev**
Pretty much the title.

### CODE REVIEW

**Comment by liondadev** [comment]
Ignore the language changes, I accidently included them in the same PR.

File: lua/darkrp_config/settings.lua [hunk 1]
```diff
@@ -1,9 +1,7 @@
 --[[-------------------------------------------------------------------------
 DarkRP config settings.
 -----------------------------------------------------------------------------
-
```
Comment (FPtje) [inline lua/darkrp_config/settings.lua:4]: Why are these newlines removed?

File: lua/darkrp_config/settings.lua [hunk 2]
```diff
@@ -525,4 +526,4 @@ Hungermod module
 -- hungerspeed <Amount> - Set the rate at which players will become hungry (2 is the default).
 GM.Config.hungerspeed = 2
 -- starverate <Amount> - How much health that is taken away every second the player is starving  (3 is the default).
-GM.Config.starverate = 3
+GM.Config.starverate = 3
```
Comment (FPtje) [inline lua/darkrp_config/settings.lua:529]: Missing newline at end of file

**Review by FPtje — CHANGES_REQUESTED** [review]
Thanks! Some commentd

**Comment by liondadev** [comment]
I dont remember removing those, ill add them back.

**Pull request merged**